salt/edx/tests/test_edxapp_running.sls

```diff
@@ -65,11 +65,6 @@ test_edxapp_{{ service }}:
     - is_running: True
 {% endfor %}
 
-test_edxapp_cms:
```

Comment:
Why remove that test?

Reply:
Because it's failing due to the fact that the supervisorctl binary isn't in the user's path and fixing that is more involved than we have time or motivation for at the moment.